.scripts/transifex/scrape.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * This is just a manual process for updating the languages available from Transifex
+ */
```

Comment:
Can you add a longer docblock description that tells what exactly the script does and why? Inline comments would also help.